### DISCUSSION TITLE
Can you scroll?

### DISCUSSION THREAD

**Discussion by NickSw1ft** [fenix-hub/godot-engine.easy-charts#76] [Q&A]
Is it possible to scroll righr/left/up/down?

**Comment by fenix-hub** [comment]
Not currently, but this feature can be implemented.
Do you have any suggestion on how you would like to be able to scroll?

**Reply by NickSw1ft** [reply]
To pan like in Desmos would be nice. Also scroll with configurable inputs, e.g. mouse wheel scrolls left-right. And would be great to be able to controll if user can or cannot scroll up-down and left-right separatly
